python/cuml/test/test_svm.py

```diff
@@ -330,16 +331,26 @@ def compare_probabilistic_svm(svc1, svc2, X_test, y_test, tol=1e-3,
     assert brier1 - brier2 <= brier_tol
 
 
-def test_svm_skl_cmp_predict_proba(n_rows=10000, n_cols=20):
+# Probabilisic SVM uses scikit-learn's CalibratedClassifierCV, and therefore
+# the input array is converted to numpy under the hood. We explicitly test for
+# all supported input types, to avoid errors like
+# https://github.com/rapidsai/cuml/issues/3090
+@pytest.mark.parametrize('in_type', ['numpy', 'numba', 'cudf', 'cupy',
+                                     'pandas', 'cuml'])
+def test_svm_skl_cmp_predict_proba(in_type, n_rows=10000, n_cols=20):
```

Comment:
not urgent now, but in the future it would be nice to make the classification dataset into a fixture with a scope of "session" so it's not regenerated repeatedly (and thus faster)... we're trying to squash down test times.